src/guides/v2.2/cloud/cdn/fastly-vcl-blocking.md

```diff
@@ -0,0 +1,100 @@
+
+**Prerequisites:**
+
+-  List of client IP addresses to block
+-  Account access and URL for the Magento Admin UI for the Staging or Production environment
+-  Fastly API credentials for Staging and Production environments
+
+## Create Edge ACL for blocking client IPs {#edge-acl}
+
+You create an Edge ACL to define the list of IP addresses to block. After creating the ACL, you can use it in a custom VCL snippet to manage access to your Staging or Production site.
+
+If you want to manage access for both Staging and Production sites, create the Edge ACL with the same name in both environments. The VCL snippet code will apply to both environments.
+
+1. Log in to the Magento Admin.
+1. Navigate to **Stores** > **Settings** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
+1. Expand the **Edge ACL** section.
+1. Click **Add ACL** to create a list. For this example, name the list "blocklist".
+1. Enter IP address values in the list. Any client IPs added to this list will be blocked access from the site.
+1. Optionally, select the **Negated** checkbox if needed.
+
+You will reference the Edge ACL by name in your VCL snippet code.
+
+## Create blocklist.json {#vcl}
+
+{:.bs-callout-info}
+This example shows advanced users how to create custom VCL code snippet to configure blocking rules that can be uploaded to the Fastly service using the Fastly API. You can also configure a blocklist or allowlist from the {{ site.data.var.ee }} Admin UI. See [Blocking](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/BLOCKING.md) in the Fastly CDN for Magento 2 module documentation.
+
+After you define the Edge ACL, you can use it to create the VCL snippet to block access to the IP addresses specified in the ACL. You can use the same VCL snippet in both Staging and Production environments, but you must upload the snippet to each environment separately.
+
+Create a `blocklist.json` file with the following VCL code in JSON format:
+
+```json
+{
+  "name": "blocklist",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "5",
+  "content": "if ( client.ip ~ blocklist) { error 403 \"Forbidden\"; }"
+}
+```
+
+Review the following parameter values and update your code snippet if necessary:
+
+-  `name`: Name for the VCL snippet. For this example, we used the name `blocklist`.
+-  `priority`: Determines the order VCL snippets call. You want to set the priority to 5 to immediately run and check for blocked IP addresses. This priority runs the snippet immediately and before any of the uploaded and default Magento VCL snippets (magentomodule) that have a priority of 50.
+-  `type`: For this VCL, we use `recv`, which places it in the `vcl_recv` subroutine, below the boilerplate VCL and above any objects.
```

Comment:
Could use an initial statement of what type is like the other list item. Is the type list short? Could list them here, if there are one or two types. Just a thought.